inspection/background.js

```diff
@@ -167,7 +162,7 @@ function sendRunTimeMessage(message) {
  * @returns current tab
  */
 async function getCurrentTab() {
-	let queryOptions = {active: true, currentWindow: true};
+	let queryOptions = { active: true, currentWindow: !0 };
```

Comment:
can we consistently use `true` instead of `!0`? I've seen this elsewhere in this file as well.

Reply:
Yes, Replaced !0 with true @mikeliucc